src/components/cards/next-event-card/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Text } from "@radix-ui/themes";
+import Image from "next/image";
+import Link from "next/link";
+
+import { INextEvent, padToStartNumber } from "@/src/constants/next-events-mock";
+
+export const NextEventCard: ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => JSX.Element = ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => {
+  function formatDateWithDayName(date: Date): string {
```

Comment:
Could you move this function to the common utils file, probably later we can reuse it.

Reply:
done